src/main/resources/assets/mwc/lang/en_us.lang

```diff
@@ -35,10 +35,11 @@ menu.example3.description=Enables example feature.
 
 # Creative Tabs
 itemGroup.resourceTab=MWC Resources
+itemGroup.otherTab=MWC Workbench's | Others
```

Comment:
For now should maybe just be called `MWC General`

Reply:
Got it